bgpd/rfapi/vnc_export_bgp.c

```diff
@@ -1177,6 +1146,121 @@ void vnc_direct_bgp_del_nve(struct bgp *bgp, struct rfapi_descriptor *rfd)
 	}
 }
 
+static void vnc_direct_add_rn_group_rd(struct bgp *bgp,
+				       struct rfapi_nve_group_cfg *rfg,
+				       struct route_node *rn,
+				       struct attr *attr,
+				       afi_t afi,
+				       struct rfapi_descriptor *irfd)
+{
+	struct prefix nhp;
+	struct bgp_info info;
+	struct attr hattr;
+	struct attr *iattr;
+
+	if (irfd == NULL && rfg->type != RFAPI_GROUP_CFG_VRF) {
+		/* need new rfapi_handle, for peer strcture
+		 * -- based on vnc_add_vrf_prefi */
+		assert (rfg->rfd == NULL);
+
+		if (!rfg->rt_export_list || !rfg->rfapi_import_table) {
+			vnc_zlog_debug_verbose("%s: VRF \"%s\" is missing RT import/export configuration.\n",
+					       __func__, rfg->name);
+			return;
+		}
+		if (!rfg->rd.prefixlen) {
+			vnc_zlog_debug_verbose("%s: VRF \"%s\" is missing RD configuration.\n",
+					       __func__, rfg->name);
+			return;
+		}
+		if (rfg->label > MPLS_LABEL_MAX) {
+			vnc_zlog_debug_verbose("%s: VRF \"%s\" is missing defaul label configuration.\n",
+					       __func__, rfg->name);
+			return;
+		}
+
+		irfd = XCALLOC(MTYPE_RFAPI_DESC,
+			      sizeof(struct rfapi_descriptor));
+		irfd->bgp = bgp;
+		rfg->rfd = irfd;
+		/* leave most fields empty as will get from (dynamic) config
```

Comment:
comment style is to have a open /* on it's own line if multiple lines of comment